apps/api/webpack.dev.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
       }
     ]
   },
-  plugins: [new NodemonPlugin(), new webpack.IgnorePlugin({ resourceRegExp: /^pg-native$/ })],
+  plugins: [new NodemonPlugin({ watch: [".env.local"] }), new webpack.IgnorePlugin({ resourceRegExp: /^pg-native$/ })],
```

Comment:
Wasn't that not working?

Reply:
Yes, need to add more dirs here for it to work properly I think